index.bs

```diff
@@ -796,11 +797,11 @@ option. If <code><a for="underlying source">type</a></code> is set to <code>unde
   </code></pre>
 </div>
 
-<h5 id="rs-asynciterator" method for="ReadableStream">getIterator({ <var>preventCancel</var> = false } = {})</h5>
+<h5 id="rs-get-iterator" method for="ReadableStream">getIterator({ <var>preventCancel</var> } = {})</h5>
```

Comment:
I didn't like the `= false` inside the method signature, as that also ended up in the table of contents. `pipeTo` and `pipeThrough` don't specify any default values for individual properties either, so this is more consistent. 🙂

Reply:
Agreed.